packages/api/CHANGELOG.md

```diff
@@ -3,6 +3,27 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [UNRELEASED]
+
+## [1.0.3](https://github.com/cennznet/api.js/compare/release/1.0.2...release/1.0.3) (13/07/2020)
+
+**Note:** Version bump only
+
+## [1.0.2](https://github.com/cennznet/api.js/compare/release/1.0.1...release/1.0.2) (15/05/2020)
+
+**Note:** Version bump only
+
+## [1.0.1](https://github.com/cennznet/api.js/compare/release/1.0.0...release/1.0.1) (27/03/2020)
+
+**Note:** Version bump only
+
+## [1.0.0](https://github.com/cennznet/api.js/compare/v0.20.5...release/1.0.0) (27/03/2020)
```

Comment:
changes log looks great to me.. just curious to investigate if the existing CHANGELOG was auto generated earlier with 
`RUN npx lerna version ${RELEASE_SCOPE} --yes --conventional-commits --force-publish='*' --changelog-preset @cennznet/conventional-changelog-preset`

Reply:
would be much easier than this if it is.
also I don't think we should bump packages if they don't have changes anymore